core/src/main/java/com/yahoo/ycsb/measurements/Measurements.java

```diff
@@ -165,7 +165,7 @@ public long getIntendedtartTimeNs() {
    * Report a single value of a single metric. E.g. for read latency, operation="READ" and latency is the measured
    * value.
    */
-  public void measure(String operation, int latency)
+  public void measure(String operation, long latency)
```

Comment:
why are we moving from int -> long on latencies in this patch? do we expect individual latencies to exceed ~35 minutes?